client/src/lib/data.ts

```diff
@@ -179,6 +251,41 @@ export async function readDateMarks(date: string): Promise<Mark[]> {
   return marks;
 }
 
+export async function readWeekMarks(date: string): Promise<Mark[]> {
+  const [start, end] = findWeekStartEnd(date);
+  const req = {
+    body: JSON.stringify({ start, end }),
+    headers: {
+      'content-type': 'application/json',
+      Authorization: ('Bearer ' + readToken()) as string,
+    },
+    method: 'post',
+  };
+  const res = await fetch('/api/marks/week', req);
+  if (!res.ok) throw new Error(`fetch Error: ${res.status}`);
+  const marks = (await res.json()) as Mark[];
+  return marks;
+}
+
+export async function readWeekMarksCal(
+  date: string,
+  calendarId: number
+): Promise<Mark[]> {
+  const [start, end] = findWeekStartEnd(date);
+  const req = {
+    body: JSON.stringify({ start, end }),
+    headers: {
+      'content-type': 'application/json',
+      Authorization: ('Bearer ' + readToken()) as string,
+    },
+    method: 'post',
+  };
+  const res = await fetch(`/api/marks/week/${calendarId}`, req);
+  if (!res.ok) throw new Error(`fetch Error: ${res.status}`);
+  const marks = (await res.json()) as Mark[];
+  return marks;
+}
+
 // export type UnsavedTodo = {
```

Comment:
You should delete this unused code. Now that it is in GitHub you can retrieve it from there if you need to refer back to it.